tools/daemon/daemon.h

```diff
@@ -90,7 +91,7 @@
 #define NAME_MAX 255 /**< chars in a file name */
 #define PATH_MAX 4096 /**< chars in a path name including null */
 #endif
-
+/*
```

Comment:
why?

Reply:
should have removed it. it's because we use doca syslog now :)